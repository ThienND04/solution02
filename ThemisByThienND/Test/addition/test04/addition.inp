10
5337 7272 6599 999999587
4209 3501 5661 999999188
3658 7492 5755 999999719
4931 7419 6753 999999164
3015 5526 6416 999999298
4634 7282 6715 999999254
4848 3894 6512 999999963
2383 6206 5629 999999272
5092 7518 6234 999999216
4770 4148 6775 999999165
