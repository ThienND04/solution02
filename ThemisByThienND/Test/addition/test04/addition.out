1952388
0
3170209
2023953
4677269
2201759
0
6006321
4190389
3730040
